welcome/index.adoc

```diff
@@ -55,6 +55,18 @@ ifdef::openshift-online,openshift-aro[]
 Start with **xref:../architecture/architecture.adoc#architecture-overview-architecture[Architecture]**.
 endif::[]
 
+ifdef::microshift[]
+Start with xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}] and
```

Comment:
We aren't supposed to hard wrap lines. Would you mind undoing the hard wraps here before we merge?